android/build.gradle

```diff
@@ -383,9 +384,9 @@ android {
 
   // We don't compress these extensions in assets/ because our random FileReader can't read zip-compressed files from apk.
   // TODO: Load all minor files via separate call to ReadAsString which can correctly handle compressed files in zip containers.
-  aaptOptions {
+  androidResources {
+    ignoreAssetsPattern '!.svn:!.git:!.DS_Store:!*.scc:.*:<dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~'
```

Comment:
I tried this change before and it didn't work. Does it work for you?

Reply:
Yes.